formats/developers/compression.txt

```diff
@@ -0,0 +1,170 @@
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+| **Size (7-zipped OME-XML)**                                 | 239,052 bytes         | 4,551,263 bytes         | 108,708,700 bytes        |
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+| **Size (OME-TIFF)**                                         | 531,384 bytes         | 10,499,384 bytes        | 225,874,680 bytes        |
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+| **Size (OME-TIFF with LZW)**                                | 273,190 bytes         | 4,998,148 bytes         | 118,517,497 bytes        |
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+| **Size (zipped OME-TIFF)**                                  | 235,764 bytes         | 4,446,727 bytes         | 105,389,599 bytes        |
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+| **Size (zipped OME-TIFF with LZW)**                         | 264,875 bytes         | 4,937,246 bytes         | 116,418,287 bytes        |
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+| **Size (7-zipped OME-TIFF)**                                | 209,593 bytes         | 3,891,846 bytes         | 93,939,055 bytes         |
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+| **Size (7-zipped OME-TIFF with LZW)**                       | 264,292 bytes         | 4,950,897 bytes         | 116,567,097 bytes        |
++-------------------------------------------------------------+-----------------------+-------------------------+--------------------------+
+
+.. _tubhiswt-2D.zip: http://downloads.openmicroscopy.org/images/OME-TIFF/2015-01/tubhiswt-2D.zip
+.. _tubhiswt-3D.zip: http://downloads.openmicroscopy.org/images/OME-TIFF/2015-01/tubhiswt-3D.zip
+.. _tubhiswt-4D.zip: http://downloads.openmicroscopy.org/images/OME-TIFF/2015-01/tubhiswt-4D.zip
+
+
+Efficiency of planar access
+---------------------------
+
+The following table compiles our results with average plane size
+computed from the numbers above, and briefly summarizes each format's
+ability to efficiently access individual image planes. We have not
+performed benchmarks involving individual planar access for each
+format—mostly because for many of these formats (especially zip, gzip
+and 7-zip) it is quite impractical to attempt efficient access to
+individual planes.
+
```

Comment:
This table isn't really quantifying the time/space tradeoffs.  The average plane size isn't telling me much about

- cpu overhead
- disk overhead
- average time-per-plane

While we're saying that "raw pixels" are the best since they can be accessed directly, this isn't taking disk bandwidth limitations into account.  When compressed with a very efficient-to-decompress method, and this doesn't include gzip, 7zip or lzw, it can be faster than uncompressed reads.